mvp.feature

```diff
@@ -60,54 +60,54 @@ Feature: User with no Author identity in the Authority Register for Personas see
+      | Data from DC meta tags                                                   |
+      | Data from title tag                                                      |
+  # There is a logical hierarchy in which these should be requested/used
+  # The intention of this step is twofold: 1) NVA knows some useful metadata about the resource, 2) researcher knows that they are linking to the right resource
 
   Scenario: User registers initial metadata for a Publication based on a Link
     Given that the user begins registering a Link
     And they see that the title metadata for the Link look-up is correct
     When they click Start
-    And the Publication metadata registration page is loaded 
+    And the Publication metadata registration page is loaded
     Then they see that the Beskrivelse tab is pre-populated with suggested metadata values (if available in the input data from the link) for
-        | Title |
-        | Alternative title(s) |
-        | Abstract |
-        | Alternative abstract(s) |
-        | Description |
-        | Publication date |
-        | NPI-fagfelt |
-        | Keywords |
-        | Primary language for publication |
+      | Title                            |
+      | Alternative title(s)             |
+      | Abstract                         |
+      | Alternative abstract(s)          |
+      | Description                      |
+      | Publication date                 |
+      | NPI-fagfelt                      |
+      | Keywords                         |
+      | Primary language for publication |
     And they they see that the Reference tab is pre-populated with suggested metadata values (if available in the input data from the link) for
-        | Publication type |
-        | Link (the link that was provided by the user) |
-        | The unmapped text for Journal |
-        | Volume |
-        | Issue |
-        | Page from | 
-        | Page to |
-        | Article number |
-        | Peer-review |          
+      | Publication type                              |
+      | Link (the link that was provided by the user) |
+      | The unmapped text for Journal                 |
+      | Volume                                        |
+      | Issue                                         |
+      | Page from                                     |
+      | Page to                                       |
+      | Article number                                |
+      | Peer-review                                   |
     And they they see that the Contributors tab Authors section is pre-populated with suggested metadata values (if available in the input data from the link) for <Author>, <Institution> and <CorrespondingAuthor>
     And they they see that the Contributors tab Contributors section is pre-populated with suggested metadata values (if available in the input data from the link) for <ContributionType>, <Name> and <Institution>
```

Comment:
Vi må være konsistent med terminologien. Er det organization eller institution? Er det det samme, eller finnes det en forskjell? Er det forskjellige termer for forfattere og bidragsytere?